src/pathfora.js

```diff
@@ -4019,6 +4033,27 @@
       return core.prepareABTest(config);
     };
 
+    this.reinitializePrioritizedWidgets = function () {
+      if (core.prioritizedWidgets.length > 0) {
+        var highest = [core.prioritizedWidgets[0]];
+
+        for (var i = 1; i < core.prioritizedWidgets.length; i++) {
```

Comment:
I one-uped you and just used the array [sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort?v=example) method! :)